### PR TITLE
Certificates: admin_details rename

### LINKED ISSUE #8718: Add/rename AdministratorContacts property on CertificateIssuer
On `CertificateIssuer`, make sure the property that returns a list of type `AdministratorContact` is named `AdministratorContacts`.

### PR DESCRIPTION
**Pull request by iscai-msft**
fixes #8718

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py [hunk 1]
```diff
@@ -886,27 +887,27 @@ def list_properties_of_certificate_versions(
...
                     email_address=admin_detail.email,
                     phone=admin_detail.phone,
                 )
-                for admin_detail in admin_details
+                for admin_detail in admin_contacts
             )
         else:
-            admin_details_to_pass = admin_details
-        if organization_id or admin_details:
+            admin_contacts_to_pass = admin_contacts
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py]: This confuses me. The overall intent seems to be, if `admin_contacts` is a list, translate it to a list of `AdministratorDetails`. But if we don't get something iterable, the `else` obtains and we pass along whatever unexpected argument we got?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py [hunk 2]
```diff
@@ -864,27 +864,27 @@ def create_issuer(self, issuer_name, provider, **kwargs):
...
+        if admin_contacts and admin_contacts[0]:
+            admin_contacts_to_pass = [
                 self._client.models.AdministratorDetails(
                     first_name=admin_detail.first_name,
                     last_name=admin_detail.last_name,
                     email_address=admin_detail.email,
                     phone=admin_detail.phone,
                 )
-                for admin_detail in admin_details
+                for admin_detail in admin_contacts
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: didn't change those for client, thanks for catching!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py [hunk 3]
```diff
@@ -929,27 +929,27 @@ def update_issuer(self, issuer_name, **kwargs):
...
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: thanks so much!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py [hunk 4]
```diff
@@ -47,14 +47,14 @@ def __repr__(self):
         )[:1024]
 
     @classmethod
-    def _from_admin_details_bundle(cls, admin_details_bundle):
+    def _from_admin_contacts_bundle(cls, admin_contacts_bundle):
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py]: thanks for catching, i got too trigger happy in models

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py [hunk 5]
```diff
@@ -1163,21 +1163,21 @@ def __repr__(self):
     def _from_issuer_bundle(cls, issuer_bundle):
         # type: (models.IssuerBundle) -> CertificateIssuer
         """Construct a CertificateIssuer from an autorest-generated IssuerBundle"""
-        admin_details = []
-        admin_details_service = (
+        admin_contacts = []
+        admin_contacts_service = (
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py]: makes sense, thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py [hunk 6]
```diff
@@ -1163,21 +1163,21 @@ def __repr__(self):
...
             issuer_bundle.organization_details.admin_details if issuer_bundle.organization_details else None
         )
-        if admin_details_service:
+        if admin_contacts_service:
             # pylint:disable=protected-access
-            for admin_detail in admin_details_service:
-                admin_details.append(AdministratorContact._from_admin_details_bundle(admin_detail))
+            for admin_contact in admin_contacts_service:
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py]: makes sense, thanks so much!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py [hunk 7]
```diff
@@ -886,27 +887,27 @@ def list_properties_of_certificate_versions(
...
-        admin_details = kwargs.pop("admin_details", None)
+        admin_contacts = kwargs.pop("admin_contacts", None)
 
         if account_id or password:
             issuer_credentials = self._client.models.IssuerCredentials(account_id=account_id, password=password)
         else:
             issuer_credentials = None
-        if admin_details and admin_details[0]:
-            admin_details_to_pass = list(
+        if admin_contacts and admin_contacts[0]:
```
[the remark this answers is not in the document]
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py]: yeah this check doesn't make sense as a whole, I'm going to just check to see if it exists and if not, just pass None

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py (hunk 1 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py]: yeah this check doesn't make sense as a whole, I'm going to just check to see if it exists and if not, just pass None

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py [hunk 8]
```diff
@@ -870,21 +870,21 @@ def create_issuer(self, issuer_name, provider, **kwargs):
...
+                    email_address=contact.email,
+                    phone=contact.phone,
                 )
-                for admin_detail in admin_contacts
+                for contact in admin_contacts
             ]
         else:
-            admin_contacts_to_pass = admin_contacts
+            admin_details = None
         if organization_id or admin_contacts:
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: `if organization_id or admin_details`? Consider what happens when we get `admin_contacts=[]`.

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py [hunk 9]
```diff
@@ -47,7 +47,7 @@ def __repr__(self):
         )[:1024]
 
     @classmethod
-    def _from_admin_contacts_bundle(cls, admin_contacts_bundle):
+    def _from_admin_details(cls, admin_contacts_bundle):
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py]: ~admin_contacts_bundle~ admin_details?

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py [hunk 10]
```diff
@@ -938,12 +938,12 @@ def update_issuer(self, issuer_name, **kwargs):
         if admin_contacts and admin_contacts[0]:
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: Comment about consequences of `admin_contacts` being a rando iterable applies here as well.

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py [hunk 11]
```diff
@@ -831,12 +831,12 @@ def list_properties_of_certificate_versions(
         if admin_contacts and admin_contacts[0]:
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py]: ... and here

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py (hunk 10 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: rip, doing that now

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py (hunk 11 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/_client.py]: thanks :'(

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py (hunk 8 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: thanks so much for catching this!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py (hunk 9 above)
Comment (iscai-msft) [reply sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py]: thanks!

File: sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py [hunk 12]
```diff
@@ -935,8 +935,8 @@ def update_issuer(self, issuer_name, **kwargs):
             issuer_credentials = self._client.models.IssuerCredentials(account_id=account_id, password=password)
         else:
             issuer_credentials = None
-        if admin_contacts and admin_contacts[0]:
-            admin_contacts_to_pass = [
+        if admin_contacts:
```
Comment (chlowell) [inline sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py]: If `admin_contacts` defaults to an empty list, you don't need this if/else.

**Pull request merged**